integration-tests/http/pom.xml

```diff
@@ -122,13 +135,30 @@
                                     <goal>native-image</goal>
                                 </goals>
                                 <configuration>
-                                    <reportErrorsAtRuntime>false</reportErrorsAtRuntime>
+                                    <!--
+                                        With GraalVM Version 19.3.1 CE this test fails with:
+
+                                            Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered
+                                            while parsing com.oracle.svm.reflect.ClassLoader_defineClass_c6c343b4d6dc22ca64eb2d8503b13ac9c340dcb3.invoke(java.lang.Object, java.lang.Object[])
+
+                                            Parsing context:
+                                                parsing java.lang.reflect.Method.invoke(Method.java:498)
+                                                parsing org.apache.camel.support.ObjectHelper.invokeMethod(ObjectHelper.java:182)
+                                                parsing org.apache.camel.main.BaseMainSupport.loadConfigurations(BaseMainSupport.java:467)
+                                                parsing org.apache.camel.main.BaseMainSupport.postProcessCamelContext(BaseMainSupport.java:522)
+
+                                        but it looks like the stack trace is unrelated to the actual error as that method
+                                        is invoked also in other tests hence I've enable reporting error at runtime to
+                                        get the test succeed.
+
+                                        TODO: investigate what is the real issue
+                                    -->
+                                    <reportErrorsAtRuntime>true</reportErrorsAtRuntime>
```

Comment:
I don't think we should have that ones in this branch because it only hides the errors and we need to be able to diagnose them.

Reply:
we just need to revert my commit

Reply:
Yeah, I think we should do that. It will it easier to reproduce the actual problem

Reply:
do it :)